Dependencies/xr/Include/XR.h

```diff
@@ -195,6 +208,7 @@ namespace xr
             Size GetWidthAndHeightForViewIndex(size_t viewIndex) const;
             void SetDepthsNearFar(float depthNear, float depthFar);
             void SetPlaneDetectionEnabled(bool enabled)const;
+            bool SetFeaturePointCloudEnabled(bool enabled)const;
```

Comment:
If this can non-fatally fail, should we call `TrySet...(bool)`?

Reply:
Also nit on this and line above: space before `const`.

Reply:
Good point will rename to TrySet...